Remotion/Web/Development.WebTesting/Schemas/WebTestingConfiguration.xsd

```diff
@@ -26,6 +26,7 @@
       <xs:attribute name="webApplicationRoot" type="c:RequiredStringType" use="required" />
       <xs:attribute name="screenshotDirectory" type="c:RequiredStringType" />
       <xs:attribute name="logsDirectory" type="c:RequiredStringType" default="." />
+      <xs:attribute name="chromiumDisableSecurityWarningsBehavior" type="c:ChromiumDisableSecurityWarningsBehavior" default="Fail" />

```

Comment:
It's already named `Chomium` here, perhaps we should leave `Chrome` for now and only rename to `Chromium` once `Edge` support is added.